packages/components/Link/src/Link.tsx

```diff
@@ -43,7 +43,12 @@ export const Link = compose<LinkType>({
       // This is a workaround for the issue. Once those issues are resolved, supportsA11yTextInText can be removed.
       const supportsA11yTextInText = Platform.OS !== 'android';
 
-      return supportsA11yTextInText && (inline || mergedProps.selectable) ? (
+      // MacOS Text component doesn't handle interaction events like hover etc.
```

Comment:
I thought it did? @amgleitman

Reply:
It's possible that this was fixed later but this is what we have in our downstream code right now.